roles/ovirt-repositories/README.md

```diff
@@ -0,0 +1,69 @@
+
+oVirt Repositories
+==================
+
+The `oVirt.repositories` role is used to set the repositories required for oVirt engine installation.
+By default it copies content of /etc/yum.repos.d/ to /tmp/repo-backup-{{timestamp}}.
+
+
+Requirements
+------------
+
+ * oVirt Python SDK version 4
+ * Ansible version 2.4
+
+Role Variables
+--------------
+
+| Name                                       | Default value         |  Description                              |
+|--------------------------------------------|-----------------------|-------------------------------------------|
+| ovirt_repositories_ovirt_rpm_repo          | UNDEF                 | URL of RPM package with repository files. |
```

Comment:
Wouldn't `ovirt_repositories_ovirt_release_rpm` be more understandable? And also description could be `URL of oVirt release package, which contains required repositories configuration` ...

Reply:
Fixed.